File: plugins/OrchidPosts/src/main/java/com/eden/orchid/posts/PostsResourceSource.java
package com.eden.orchid.posts;

import com.eden.orchid.api.OrchidContext;
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource;

import javax.inject.Inject;
import javax.inject.Singleton;

@Singleton
public class PostsResourceSource extends PluginResourceSource {

    @Inject
    public PostsResourceSource(OrchidContext context) {
        super(context, 20);
    }
}

File: plugins/OrchidPosts/src/main/kotlin/com/eden/orchid/posts/PostsResourceSource.kt
package com.eden.orchid.posts

import com.eden.orchid.api.OrchidContext
import com.eden.orchid.api.resources.resourceSource.PluginResourceSource

import javax.inject.Inject
import javax.inject.Singleton

@Singleton
class PostsResourceSource @Inject
constructor(context: OrchidContext) : PluginResourceSource(context, 20)

